File: tfplus/pangu/python/ops/pangufs_ops.py
# Copyright 2019 The TF-plus Authors. All Rights Reserved.

"""Pangu File System Support

Pangu is an Distributed FileSystem provided by Alibaba. This module
implements a filesystem on top of it.
"""

from __future__ import absolute_import
from __future__ import division
from __future__ import print_function


from tfplus.common import _load_library
_load_library("_pangu_ops.so")

File: tfplus/pangu/ops/pangufs_ops.cc
// Copyright 2019 The TF-plus Authors. All Rights Reserved.

#include "tensorflow/core/platform/env.h"

#include "tfplus/pangu/kernels/pangufs/pangu_file_system.h"

namespace tensorflow {

REGISTER_FILE_SYSTEM("pangu", PanguFileSystem);

}  // namespace tensorflow
